functional-tests/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "ES2019",
+    "module": "commonjs",
```

Comment:
Curios, if we're using TS why are we still referencing commonjs?

Reply:
It's the module output format, as the transipled JS is used within Node, so using commonjs is what is needed to run properly there. See https://www.typescriptlang.org/tsconfig#module